templates/zerver/integrations/errbot.md

```diff
@@ -25,23 +27,42 @@ Run your favorite chatbot in Zulip!
         BOT_LOG_FILE = r'<path/to/your/errbot/logfile.log>'
         BOT_LOG_LEVEL = logging.INFO
 
-        BOT_IDENTITY = {  # Fill this with the corresponding values in your bot's `.zuliprc`
```

Comment:
Removed this mention of .zuliprc.
Inserted the link to the bots page separately in a tip, since we can't link inside this code block.